third_party/2and3/flask/blueprints.pyi

```diff
@@ -34,7 +34,8 @@ class Blueprint(_PackageBoundObject):
     static_url_path: Optional[str] = ...
     deferred_functions: Any = ...
     url_values_defaults: Any = ...
-    def __init__(self, name: str, import_name: str, static_folder: Optional[str] = ..., static_url_path: Optional[str] = ..., template_folder: Optional[str] = ..., url_prefix: Optional[str] = ..., subdomain: Optional[str] = ..., url_defaults: Optional[Any] = ..., root_path: Optional[str] = ...) -> None: ...
+    cli_group: Union[Optional[str], object] = ...
+    def __init__(self, name: str, import_name: str, static_folder: Optional[str] = ..., static_url_path: Optional[str] = ..., template_folder: Optional[str] = ..., url_prefix: Optional[str] = ..., subdomain: Optional[str] = ..., url_defaults: Optional[Any] = ..., root_path: Optional[str] = ..., cli_group: Union[Optional[str], object] = ...) -> None: ...
```

Comment:
In an argument position, `Union[Optional[str], object]` is just object, so this won't catch incorrect types being passed to this arg. One way to do that could be to add a private empty `class _Sentinel(object): pass` to this file and use it in the type.